index.bs

```diff
@@ -102,18 +100,18 @@ CompressionStream includes GenericTransformStream;
 The <dfn interface>CompressionStream(format)</dfn> constructor, when invoked, must run these steps:
     1. If <dfn>*format*</dfn> is unsupported in CompressionStream, then throw a TypeError.
     1. Let *cs* be a new CompressionStream object.
-    1. Let *transformAlgorithm* be an algorithm which takes a *chunk* argument and runs the <dfn>compress and enqueue a chunk</dfn> algorithm with *cs* and *chunk*.
+    1. Let *transformAlgorithm* be an algorithm which takes a *chunk* argument and runs the <a href="#compress-and-enqueue"><dfn>compress and enqueue a chunk</dfn></a> algorithm with *cs* and *chunk*.
```

Comment:
You should remove the `<dfn>` and `</dfn>` here. Bikeshed normally finds the correct target for `<a>` tags automatically based on the text, so you don't need to specify `href`.